WcaOnRails/db/migrate/20160731000001_create_committees.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class CreateCommittees < ActiveRecord::Migration
+  def change
+    create_table :committees do |t|
+      t.string :name, limit: 50, null: false
+      t.string :slug, limit: 50, null: false
```

Comment:
@pingskills there are many migrations in this PR using limit and the structure.sql, please fix them =)